2022-06-27 20:52:40,897 DEBUG    root            Loaded Command Group: ['gcloud', 'config']
2022-06-27 20:52:40,899 DEBUG    root            Loaded Command Group: ['gcloud', 'config', 'config_helper']
2022-06-27 20:52:40,902 DEBUG    root            Running [gcloud.config.config-helper] with arguments: [--format: "json", --quiet: "True"]
2022-06-27 20:52:40,918 DEBUG    google.auth.transport.requests Making request: GET http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/julio31529@gmail.com/?recursive=true
2022-06-27 20:52:40,921 DEBUG    urllib3.connectionpool Starting new HTTP connection (1): metadata.google.internal:80
2022-06-27 20:52:40,924 DEBUG    urllib3.connectionpool http://metadata.google.internal:80 "GET /computeMetadata/v1/instance/service-accounts/julio31529@gmail.com/?recursive=true HTTP/1.1" 200 None
2022-06-27 20:52:40,925 DEBUG    google.auth.transport.requests Making request: GET http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/julio31529@gmail.com/token
2022-06-27 20:52:40,927 DEBUG    urllib3.connectionpool Resetting dropped connection: metadata.google.internal
2022-06-27 20:52:40,930 DEBUG    urllib3.connectionpool http://metadata.google.internal:80 "GET /computeMetadata/v1/instance/service-accounts/julio31529@gmail.com/token HTTP/1.1" 200 None
2022-06-27 20:52:40,946 INFO     root            Display format: "default json"
2022-06-27 20:52:40,947 INFO     ___FILE_ONLY___ {
  "configuration": {
    "active_configuration": "default",
    "properties": {
      "accessibility": {
        "screen_reader": "True"
      },
      "component_manager": {
        "disable_update_check": "True"
      },
      "compute": {
        "gce_metadata_read_timeout_sec": "30"
      },
      "core": {
        "account": "julio31529@gmail.com",
        "disable_prompts": "True",
        "disable_usage_reporting": "True",
        "print_unhandled_tracebacks": "true"
      },
      "metrics": {
        "environment": "cloudcode.vscode",
        "environment_version": "1.18.3"
      }
    }
  },
  "credential": {
    "access_token": "ya29.a0ARrdaM_nNqZuRbkus3dw6D9kFZwxl9dWCt5GhR1AI4CCTScQVeKyZXGbJj4trH-FobQKXUUc7J8Mc5VjyItCozvAVxvVcASgni-Sf-o0F3raHQqKnXfKrgcxT-Xz1pIfECOiErx6oWaRGcpZS6tPMIbOF5rk_4i5rfbicdiHfBzTNqjseR_y1Mj8KeCTFZmZoB3Q9klBPmVm1yUbNpsW0Lu_G-beStqCEEkEdQ15W9i1EMUjdEWZ5d_VdrVKMmwuvt_JhvE",
    "id_token": "eyJhbGciOiJSUzI1NiIsImtpZCI6IjJiMDllNzQ0ZDU4Yzk5NTVkNGYyNDBiNmE5MmY3YjM3ZmVhZDJmZjgiLCJ0eXAiOiJKV1QifQ.eyJpc3MiOiJhY2NvdW50cy5nb29nbGUuY29tIiwiYXpwIjoiNjE4MTA0NzA4MDU0LTlyOXMxYzRhbGczNmVybGl1Y2hvOXQ1Mm4zMm42ZGdxLmFwcHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tIiwiYXVkIjoiNjE4MTA0NzA4MDU0LTlyOXMxYzRhbGczNmVybGl1Y2hvOXQ1Mm4zMm42ZGdxLmFwcHMuZ29vZ2xldXNlcmNvbnRlbnQuY29tIiwic3ViIjoiMTAwMjMxNjkyOTA0MDk4MDE4NzEwIiwiZW1haWwiOiJqdWxpbzMxNTI5QGdtYWlsLmNvbSIsImVtYWlsX3ZlcmlmaWVkIjp0cnVlLCJhdF9oYXNoIjoid3V0QmoyY09GcmFLM2h5SV82OVByUSIsImlhdCI6MTY1NjM2MDkzOCwiZXhwIjoxNjU2MzY0NTM4LCJqdGkiOiJkOTU5MTdiYjNlYTZhZjQwZThkMjc0ZTUzYjdhNGE1ZGUxNGJjZTdkIn0.ba0kmFJipmUIa2OO0V7anC5zuc5-Qc9HpdocJLOYATF4aLwlWOkNC_kpBWZVSfaTxiz2m01077D2HciWlK3UF7hwgQsHQJ_1CYjrhckMXzpyjOlkH6DDk7BZVPCELGFOJKhfQuSHV5ppYTyjVvnK2SKlPZO1qIiyJKQnp9_r6QbnmFZH7Psa9Y0-2bjPwuzKRgw40G-i22XlOgQhNQzbm-Jk_ryFQnez7QVylFnrTc70f5KTksaMoxtMbaCimy2U81bhnDivrNcU3Xr4JXVW4jK58GcqamL2t4XcUBfIp2aZBk8ZJnLflmD7rFSEyCN1TEb1xTdWhatDsj6BirTGHQ",
    "token_expiry": "2022-06-27T21:15:36Z"
  },
  "sentinels": {
    "config_sentinel": "/home/julio31529/.config/gcloud/config_sentinel"
  }
}
2022-06-27 20:52:40,948 INFO     ___FILE_ONLY___ 

2022-06-27 20:52:40,948 DEBUG    root            [Errno 32] Broken pipe
Traceback (most recent call last):
  File "/usr/lib/google-cloud-sdk/lib/googlecloudsdk/calliope/cli.py", line 987, in Execute
    resources = calliope_command.Run(cli=self, args=args)
  File "/usr/lib/google-cloud-sdk/lib/googlecloudsdk/calliope/backend.py", line 810, in Run
    resources = display.Displayer(command_instance, args, resources,
  File "/usr/lib/google-cloud-sdk/lib/googlecloudsdk/calliope/display.py", line 491, in Display
    log.out.flush()
  File "/usr/lib/google-cloud-sdk/lib/googlecloudsdk/core/log.py", line 247, in flush
    self.__stream_wrapper.stream.flush()
BrokenPipeError: [Errno 32] Broken pipe
